google/cloud/spanner_v1/_pandas_helpers.py

```diff
@@ -0,0 +1,76 @@
+import logging
```

Comment:
Needs Apache 2.0 license header. See other files.

Reply:
Modified the header